src/components/BirdAlertList.tsx

```diff
@@ -78,6 +81,14 @@ export default function BirdAlertList() {
         setIsLoading(false)
     }
 
+    function renderSecondVolunteerElements(rescue : BirdAlert) {
+        if (rescue.secondVolunteer && rescue.twoPersonRescue) {
```

Comment:
This can have an early return if `!rescue.twoPersonRescue`. Since this is used in two components we could also break it out to a separate tiny component, but maybe not worth the hassle.